src/log.rs

```diff
@@ -4,44 +4,13 @@
 
 use crate::context::Context;
 
-#[macro_export]
-macro_rules! info {
-    ($ctx:expr,  $msg:expr) => {
-        info!($ctx, $msg,)
-    };
-    ($ctx:expr, $msg:expr, $($args:expr),* $(,)?) => {{
-        let formatted = format!($msg, $($args),*);
-        let full = format!("{file}:{line}: {msg}",
-                           file = file!(),
-                           line = line!(),
-                           msg = &formatted);
-        $ctx.emit_event($crate::EventType::Info(full));
-    }};
-}
-
-#[macro_export]
-macro_rules! warn {
```

Comment:
Since the new `warn!` macro doesn't take a `context` anymore, would be nice to adapt `log_err()` and so on, too, maybe by just letting it use the `warn!` macro. After rebasing though, because it'll create a merge conflict otherwise, and it's not urgent either.